src/items/generics.md

```diff
@@ -43,18 +43,137 @@ struct Ref<'a, T> where T: 'a { r: &'a T }
 struct InnerArray<T, const N: usize>([T; N]);
 ```
 
+The generic parameters are in scope within the item definition where they are
+declared.
+
+[References], [raw pointers], [arrays], [slices][arrays], [tuples], and
```

Comment:
Introduced in #258. If I had to guess, it is maybe trying to say is that the syntax for specifying arguments to those types does not use the same syntax that the other items (structs, fns, etc.) use. That is, instead of IDENT<…> they each have their own unique syntax for specifying arguments.

I agree, it seems a bit confusing. Any suggestions? Maybe @matthewjasper could clarify?

Reply:
Yeah, I let my brain think about it overnight. I'm betting this statement was originally in `paths.html` where it makes sense and was moved here as part of a split out. We should probably bring most of `GenericArgument` into `generics.html` and then rename `GenericArguments` to `GenericPathArguments` or something. I'll open an issue for that one.